frontend/src/components/page_content/profile-content.js

```diff
@@ -5,13 +5,18 @@ import Profile from "./profile/profile";
 import Finances from "./profile/finances";
 import Family from "./profile/family";
 import Goals from "./profile/goals.js";
+import { resetStep } from "../../redux/actions";
 import { connect } from "react-redux";
 
 class ProfileContent extends React.Component {
   onSubmit() {
     // submit form
   }
 
+  componentDidMount() {
```

Comment:
i dont think this is necessary because its reset already in the page template component